contributions/presentation/week6/ved.megha@gmail.com/readme.md

```diff
@@ -0,0 +1 @@
+#testing
```

Comment:
Only one `readme.md` can be added in a PR. Please remove this file from your PR.

Reply:
The file readme.md is deleted now.